### DISCUSSION TITLE
vc-measurements组件绘制了多个面，怎样子动态设置每个面的颜色属性？

### DISCUSSION THREAD

**Discussion by 2452615812** [zouyaoji/vue-cesium#471] [Q&A]
vc-measurements组件绘制了多个面，这些面的坐标数据都存在areaMeasurementOpts的preRenderDatas内后，想自定义这些面的颜色属性，areaMeasurementOpts.polygonOpts.appearance.options.material.fabric.uniforms.color= colorHextoRGB(“409EFF”);
     areaMeasurementOpts.primitiveOpts.appearance.options.material.fabric.uniforms.color= “409EFF”; 这样写所有面都被修改了，单独修改一个面的颜色，该怎么做？

**Comment by zouyaoji** [comment]
参考一下 https://github.com/zouyaoji/vue-cesium/discussions/206

拿到renderDatas去改
